tests/test_common.py

```diff
@@ -355,11 +358,18 @@ def __init__(self, *args, **kwargs):
             '--log_file', TestCommon.log_file
         ]
         os.environ['CLARITYSCRIPTCONFIG'] = join(self.etc_path, 'example_clarity_script.yml')
+        self.current_wd = os.curdir
 
     def setUp(self):
         argv = self.default_argv.copy()
         argv[1] = 'http://server:8080/some/extra/stuff'
         self.epp = StepEPP(argv)
+        self.current_wd = os.curdir
```

Comment:
This is defined in \_\_init\_\_ above - which one do we want?

Reply:
We want both. When subclasses are overriding `setUp` without calling `super` the one in `__init__` allows `tearDown` to still pass.

Reply:
updated